package.json

```diff
@@ -76,8 +89,11 @@
     "express": "^4.16.2",
     "express-graphql": "^0.6.11",
     "graphql-tools": "^2.18.0",
+    "husky": "^0.14.3",
```

Comment:
I think we need a `yarn.lock` update too for all these new deps.

Reply:
I'm actually not entirely sure. I was under the impression that `lint-staged` used `husky` under the hood.

https://prettier.io/docs/en/precommit.html

Reply:
So reading that article it looks like this PR is missing `package.json:scripts` tasks then... (?)

Reply:
Fixed, sorry 'bout that.